armeria-backend/zio/src/test/scala/sttp/client3/armeria/zio/ArmeriaZioHttpTest.scala

```diff
@@ -0,0 +1,19 @@
+package sttp.client3.armeria.zio
+
+import sttp.client3._
+import sttp.client3.impl.zio.ZioTestBase
+import sttp.client3.testing.{ConvertToFuture, HttpTest}
+import zio.Task
+
+class ArmeriaZioHttpTest extends HttpTest[Task] with ZioTestBase {
+
+  // FIXME(ikhoon): A request failed with `ResponseTimeoutException`.
+  //                However, "read exceptions - timeout" test never ends.
```

Comment:
I don't know why `read exceptions - timeout` test never ends.
Can I get some advice?

Reply:
does this happen only for the zio backend? and both locally and on CI?

Reply:
- It only happens for ZIO backend. 
- It is reproducible locally.

Reply:
There was a bug in closing a client factory. The problem is fixed now.